themes/keystone/src/scss/base/_variables.scss

```diff
@@ -232,10 +230,10 @@ $navigation-boxShadow:                    0 5px 10px 0 rgba(0,0,0,0.30);
 $navigation-hover_fg:                     $global-color_primary !default;
 $navigation_border:                       darken($navigation_bg, 8%)!default;
 $navigation_borderRadius:                 $global_borderRadius !default;
-$navigation_fontSize:                     $global-medium_fontSize !default;
+$navigation_fontSize:                     $global-large_fontSize !default;
```

Comment:
Just a note: why **large** replace **medium**?

Reply:
Because all font sizes decreased (before `$global-medium_fontSize` was 16px and `$global-large_fontSize` was 18px, now `$global-large_fontSize` is 16px. Since this is for mobile only I decided to increase a little bit for better visibility).